client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/rates-step/style.scss

```diff
@@ -66,6 +66,10 @@
 		// See: https://github.com/WordPress/gutenberg/pull/23706
 		border: 4px solid #d52c82;
 		box-sizing: border-box;
+
+		// Unset the translate() transform introduced in Gutenberg.
+		// See: https://github.com/WordPress/gutenberg/pull/27377
+		transform: none;
```

Comment:
Looks like there are some unwanted changes after the force-push

Reply:
haha omg, you are so fast. yes. I need to rebase from the latest develop. D: